tck/src/main/java/org/reactivestreams/tck/PublisherVerification.java

```diff
@@ -573,6 +573,32 @@ public void run(Publisher<T> pub) throws Throwable {
   }
 
   @Override @Test
+  public void optional_spec111_registeredSubscribersMustReceiveOnNextOrOnCompleteSignals() throws Throwable {
+    optionalActivePublisherTest(1, false, new PublisherTestRun<T>() {
+      @Override
+      public void run(Publisher<T> pub) throws Throwable {
+        ManualSubscriber<T> sub1 = env.newManualSubscriber(pub);
+        ManualSubscriber<T> sub2 = env.newManualSubscriber(pub);
+        //It's implementation dependant to be a multicast or unicast.
+        //So we don't know which subscriber will receive an onNext (and onComplete) and which just onComplete.
```

Comment:
Let's add a small comment that since we're testing 1.11 we can assume that either it does not support this and thing will blow up - resulting in a skipped test or that it DOES actually support multiple subscribers (so the MAY in Rule 1.11 is fulfilled). Then the sentences here are put into better context. What I mean to highlight is that it's not just the two cases that are spec wise legal, but that in THIS test we assume that it does support multiple subscribers, which means that it will do one of those things.

The test is OK, since if it fails, it will be marked as optional and skipped - however, the comment should not lead people to think directly about only those two code paths (implementors are expected to read the source sometimes).

How about we add a comment that "Since we're testing the case when the Publisher DOES support the optional multi-subscribers scenario, and decides if it handles them uni-cast or multi-cast..." and then your comment.

In general let's try to be as helpful as possible when we do such comments (perhaps not all of current ones are perfect, but let's aim for it :-))